### DISCUSSION TITLE
Add a section to the wiki about compiling / running the code from source

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13552] [Feature Requests]
I'm eager to get started with some smallish PRs that I'd like to see working before committing. I'm new to node, not so new to javascript though, but can't understand how to get the system running from source - it isn't exactly running a single index.js file.

So could a short section be added to the wiki on how to do it please. Perhaps it will help others to get started as well!

**Comment by rijkvanzanten** [comment]
Have you seen https://docs.directus.io/contributing/running-locally/#running-locally? 🙂

**Reply by sxsws** [reply]
I had not, thanks! That breaks it down into nice steps. The docs for directus are exemplary.
